public/components/player/list/list.stache

```diff
@@ -20,7 +20,7 @@
 				<tr>
 					{{#eq . editingPlayer}}
 						<td>
-							<player-edit player="{.}"
+							<player-edit player="{.}" {is-admin}="isAdmin"
```

Comment:
Does this need to get passed?  If someone isn't an admin, they shouldn't be able to even set `editingPlayer`?